recipe/recipe.yaml

```diff
@@ -23,16 +23,13 @@ requirements:
     - setuptools
   run:
     - python >=${{ python_min }}
-    - numba >=0.59.1
+    - numba >=0.60.0
     - cuda-bindings  # Includes nvrtc, nvvm, libdevice for CUDA 12, cudatoolkit for CUDA 11
```

Comment:
Do we want to pin to `>=12.9.1` based on https://github.com/NVIDIA/numba-cuda/pull/387?